packages/stable/src/api/geospatial/types.ts

```diff
@@ -269,3 +269,38 @@ export interface GeospatialCRSResponse
 export interface GeospatialSridId {
   srid: number;
 }
+
+export type GeospatialComputeFunction =
+  | {
+      stTransform: GeospatialGeometryTransformComputeFunction;
+    };
+
+export interface GeospatialGeometryTransformComputeFunction {
+  geometry: GeospatialGeometryComputeFunction;
+  srid: number;
+}
+
+export type GeospatialGeometryComputeFunction =
+  | {
+      geometryValue: GeospatialGeometryValueComputeFunction;
+    };
+
+export interface GeospatialGeometryValueComputeFunction {
+  ewkt: string;
+}
+
+export interface GeospatialComputedItem {
+  item: Map<string, any>;
+}
+
+export interface GeospatialComputedItemList {
+  items: GeospatialComputedItem[];
+}
+
+export interface GeospatialComputeRequest {
+  output: Map<string, GeospatialGeometryComputeFunction>
+}
+
+export interface GeospatialComputeResponse {
+  items: GeospatialComputedItemList;
```

Comment:
suggestion(blocking): we generally do not need `items` in our types as that adapter is taken care by the base sdk.
Need to figure out which endpoint method we've to use for compute.
We've options like `listEndpoint`, `retrieveEndpoint`, etc.